openedx/adg/common/lib/mandrill_client/client.py

```diff
@@ -15,6 +16,11 @@ class MandrillClient(object):
     """
     Mandrill class to send ADG emails
     """
+
+    # Note: Language is trimmed from the template name constant e.g.
+    # On mandrill we have 'adg-password-reset' for English and 'adg-password-reset-ar' for Arabic
+    # however, constant is defined as 'adg-password-reset'
```

Comment:
- How about making this note part of the doc string?
- Also let's use `slug` instead of `name constant`, so
```
Note: While adding here, language is trimmed from the template slug
For example on mandrill if we have ..... ...... ....... ..... .... .... ... for Arabic
here it would be defined as 'adg-password-reset'
```